src/scripts/items/HeldItem.ts

```diff
@@ -251,4 +251,11 @@ ItemList.Everstone = new HeldItem('Everstone', 10000, GameConstants.Currency.mon
         // babies
         const baby = pokemonBabyPrevolutionMap[pokemon.name];
         return baby !== undefined && pokemon.name != baby;
-    });
+    }
+);
+
+// Override Black Glasses' canUse function to factor in Squirtle evolution.
+const glassesRestrict = (ItemList.Black_Glasses as AttackBonusHeldItem).canUse;
+(ItemList.Black_Glasses as AttackBonusHeldItem).canUse = (pokemon: PartyPokemon): boolean => {
+    return pokemon.name != 'Squad Leader Squirtle' ? pokemon.name == 'Squirtle' || glassesRestrict(pokemon) : false;
+};
```

Comment:
I'm not exactly thrilled with doing it this way but the only alternatives I can think of are having `TypeRestrictedAttackBonusHeldItem` accept an optional function parameter for additional criteria that `canUse` calls, or extend `TypeRestrictedAttackBonusHeldItem` with a new class to do something similar

Reply:
I think it also feels unintuitive, although maybe that's not a bad thing, since it's an optional Pokémon.